js/foundation.interchange.js

```diff
@@ -5,6 +5,7 @@ import { MediaQuery } from './foundation.util.mediaQuery';
 import { Plugin } from './foundation.core.plugin';
 import { GetYoDigits } from './foundation.core.utils';
 
+import { Triggers } from './foundation.util.triggers';
```

Comment:
Can you remove the empty line?

Reply:
Sure can, only put it in because it was spaced like this in the other plugins